src/nvim/main.c

```diff
@@ -2057,93 +2057,66 @@ static void mainerr(int n, const char *str)
 }
 
 
-/*
- * print a message with three spaces prepended and '\n' appended.
- */
-static void main_msg(char *s)
-{
-  mch_msg("   ");
-  mch_msg(s);
-  mch_msg("\n");
-}
-
-/*
- * Print messages for "vim -h" or "vim --help" and exit.
- */
+/// Prints help message and exits
```

Comment:
Noting "vim -h" or "vim --help" is useful, especially for grepping, but also for zero-friction comprehension of what a function does.